src/main/java/com/google/collegeplanner/data/Course.java

```diff
@@ -0,0 +1,201 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.package com.google.collegeplanner.data;
+package com.google.collegeplanner.data;
+
+import java.lang.InstantiationException;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+public class Course {
+  /*
+   * courseId represents the name of the course that
+   * can be found on the course catalog. Example, ENGL101
+   */
+  private String courseId;
+  /*
+   * name represents the full name of the course.
+   * Example, Academic Writing
+   */
+  private String name;
+  /*
+   * semester represents the 4 digit year followed by
+   * the two digit month when the course starts. Example, 202008
+   */
+  private String semester;
+  /*
+   * credits represents the number of credits the course is worth.
+   */
+  private int credits;
+  /*
+   * deptId represents the 4 character ID of the
+   * department that the class falls into. Example, ENGL
+   */
+  private String deptId;
```

Comment:
Google style guide tends to discourages shorthands like this in favor of clarity of full english words when possible (except for very obvious ones like HTTP, URL, etc). Let's have this be `departmentId`.

Below, I think we also have coreqs/prereqs which can be corequisites/prerequisites, but that's slightly more borderline. Feel free to modify those if you want, but I won't block on it.